### ISSUE TITLE
the recording begin time of detector E1 output file

### ISSUE THREAD

**Issue by yuameier111** [eclipse/sumo#13521]
is there any way to set the begin time for recording in detector output file

**Comment by namdre** [comment]
No. At the moment, recording always starts at the start of the simulation. However, with careful choice of the aggregation period you may be able to just exclude a number of intervals from later analysis. Some simialr traffic measures can also be obtained with edgeData / laneData output (https://sumo.dlr.de/docs/Simulation/Output/Lane-_or_Edge-based_Traffic_Measures.html) when restricted to specific edges via attribute `edges` and then you can set the begin/end time.
